var/spack/repos/builtin/packages/py-breathe/package.py

```diff
@@ -38,6 +39,7 @@ class PyBreathe(PythonPackage):
     depends_on("py-sphinx", type=("build", "run"))
     depends_on("py-sphinx@:3", type=("build", "run"), when="@:4.20")
     depends_on("py-sphinx@3:3.2", type=("build", "run"), when="@4.21:4.32")
-    depends_on("py-sphinx@3:", type=("build", "run"), when="@4.33:")
+    depends_on("py-sphinx@3:4", type=("build", "run"), when="@4.33:")
```

Comment:
```suggestion
    depends_on("py-sphinx@3:4", type=("build", "run"), when="@4.33")
```
Otherwise this constraint will also be applied to 4.34.0

Reply:
Good point... probably
```
depends_on("py-sphinx@3:4", type=("build", "run"), when="@4.33.0:4.33")
```
because 4.33.1 is not in `4.33` (oh spack versions, how I love you so)

Reply:
Nah, `@4.33` works. At least it works with clingo, don't care about the old concretizer.

Reply:
what seriously? I was convinced 4.33 was a specific version and `4.33.0:4.33` was a ranged version...

Reply:
It used to be in the old concretizer. I think it still is on the command line.

Reply:
I think the command line has scarred me for life :(